src/selectors/columnWidths.js

```diff
@@ -162,10 +171,25 @@ function groupColumns(columnProps) {
     container.push(columnProp);
   });
 
+  forEach(columnGroupProps, (columnProp, index) => {
+    let container = scrollableColumnGroups;
+    if (columnProp.fixed) {
+      container = fixedColumnGroups;
+    } else if (columnProp.fixedRight) {
+      container = fixedRightColumnGroups;
+    }
+    columnGroupIndex[index] = container.length;
```

Comment:
this line is how we map actual index to the group index, which is used in the reducer (explained above)